linux_sql/scripts/sql/ddl.sql

```diff
@@ -0,0 +1,60 @@
+postgres=# \c host_agent;
+
+-- Create a host_info table if it does not already exist
+CREATE TABLE IF NOT EXISTS PUBLIC.host_info 
+  ( 
+     id               SERIAL NOT NULL, 
+     hostname         VARCHAR NOT NULL, 
+     cpu_number       INT2 NOT NULL, 
+     cpu_architecture VARCHAR NOT NULL, 
+     cpu_model        VARCHAR NOT NULL, 
+     cpu_mhz          FLOAT8 NOT NULL, 
+     l2_cache         INT4 NOT NULL, 
+     "timestamp"      TIMESTAMP NULL, 
+     total_mem        INT4 NULL, 
+     CONSTRAINT host_info_pk PRIMARY KEY (id), 
+     CONSTRAINT host_info_un UNIQUE (hostname) 
```

Comment:
Is there a reason you chose to define these constraints at a table level rather than column level?